syntax/LaTeX.tmLanguage.json

```diff
@@ -817,7 +817,7 @@
       ]
     },
     {
-      "begin": "(?:\\s*)((\\\\)begin)(\\{)(tabular[xy*]?|xltabular|longtable)(\\})(\\s*\\n)?",
+      "begin": "(?:\\s*)((\\\\)begin)(\\{)(tabular[xy*]?|xltabular|longtable|(long)?tabu)(\\})(\\s*\\n)?",
```

Comment:
The added item should use a non-capturing group `(?:long)?tabu`, otherwise the fifth group will not reference the closing `}`

https://github.com/James-Yu/LaTeX-Workshop/blob/a1210cdbb63e2705d9f1c79b82f2dd8b28933fa3/syntax/LaTeX.tmLanguage.json#L834-L836

Reply:
Sorry, missed that! I've changed the added group to a non-capturing group.